pkgs/top-level/haskell-packages.nix

```diff
@@ -108,19 +108,19 @@ in {
           packages.ghc8107BinaryMinimal
```

Comment:
You can drop the aarch64-darwin condition above, since stdenv.isAarch64 will also catch that.

Reply:
Yes, that makes sense.